src/motor-operations.h

```diff
@@ -45,6 +46,17 @@ struct LinearSegmentSteps {
   int steps[BEAGLEG_NUM_MOTORS]; // Steps for axis. Negative for reverse.
 };
 
+// Struct used to return data about the currently executed steps
+// and the status of the auxes.
+struct PhysicalStatus {
+  // Absolute position in steps
+  int pos_steps[BEAGLEG_NUM_MOTORS];
+  // Auxes status
+  unsigned short aux_bits;
+  // A Tag? It may be useful to add a tag to map back
```

Comment:
Yes, that sounds like a good idea in the future. We would need to provide that tag when we Enqueue() the LinearSegmentStep.
(something for future work).